tests/src/FunctionalJavascript/GroupsTagsSubmissionTest.php

```diff
@@ -57,57 +55,28 @@ public function testSubmitWebform() {
     $this->assertPageNoErrorMessages();
 
     $this->assertSession()->waitForField('First Name');
-
-    // The Default Existing Contact Element behaviour is: load logged in User
-    // The test here is to check if the fields on the form populate with Contact details belonging to the logged in User:
-    $this->assertSession()->fieldValueEquals('First Name', 'Maarten');
-    $this->assertSession()->fieldValueEquals('Last Name', 'van der Weijden');
+    $this->getSession()->getPage()->fillField('First Name', 'Frederick');
+    $this->getSession()->getPage()->fillField('Last Name', 'Pabst');
+    $this->getSession()->getPage()->fillField('Email', 'frederick@pabst.io');
 
     $this->getSession()->getPage()->checkField('Volunteer');
     $this->assertSession()->checkboxChecked("Volunteer");
     $this->htmlOutput();
 
     $this->getSession()->getPage()->pressButton('Submit');
-    // ToDo: Fix Notice on GitHub tests Notice: Undefined index: #form_key in Drupal\webform_civicrm\Utils->wf_crm_enabled_fields() (line 477 of /home/runner/work/webform_civicrm/webform_civicrm/src/Utils.php).
-    // $this->assertPageNoErrorMessages();
-    $this->htmlOutput();
-    $this->assertSession()->pageTextContains('New submission added to CiviCRM Webform Test.');
-
-    $utils = \Drupal::service('webform_civicrm.utils');
-    $api_result = $utils->wf_civicrm_api('Contact', 'get', [
-      'sequential' => 1,
-      'return' => ["tag"],
-      'contact_id' => 2,
-    ]);
-
-    $this->assertEquals('Volunteer', $api_result['values'][0]['tags']);
-
-    // Ok let's go back to the Webform and see it is loading the previously selected Tag properly
-    $this->drupalGet($this->webform->toUrl('canonical'));
     $this->assertPageNoErrorMessages();
-    $this->assertSession()->waitForField('Volunteer');
-    $this->assertSession()->checkboxChecked('Volunteer');
-    $this->assertSession()->checkboxNotChecked('Company');
-
-    // Ok clear the Tag:
-    $this->getSession()->getPage()->uncheckField('Volunteer');
-    $this->assertSession()->checkboxNotChecked('Volunteer');
     $this->htmlOutput();
-    $this->getSession()->getPage()->pressButton('Submit');
-    // ToDo: Fix Notice on GitHub tests Notice: Undefined index: #form_key in Drupal\webform_civicrm\Utils->wf_crm_enabled_fields() (line 477 of /home/runner/work/webform_civicrm/webform_civicrm/src/Utils.php).
-    // $this->assertPageNoErrorMessages();
     $this->assertSession()->pageTextContains('New submission added to CiviCRM Webform Test.');
 
     $utils = \Drupal::service('webform_civicrm.utils');
     $api_result = $utils->wf_civicrm_api('Contact', 'get', [
       'sequential' => 1,
       'return' => ["tag"],
-      'contact_id' => 2,
+      'contact_id' => 4,
```

Comment:
Can we use a $contactId by retrieving it from api? similar to what is done [here](https://github.com/colemanw/webform_civicrm/blob/8.x-5.x/tests/src/FunctionalJavascript/CustomFieldSubmissionTest.php#L268-L273? ) .

Reply:
working on the test file so will merge this and include the above fix in my PR.